plugins/generic/pln/classes/DepositDAO.inc.php

```diff
@@ -124,7 +124,7 @@ function &getNeedPackaging($journalId) {
 			array(
 				(int) $journalId,
 				(int) PLN_PLUGIN_DEPOSIT_STATUS_PACKAGED,
-				(int) PLN_PLUGIN_DEPOSIT_STATUS_LOCAL_FAILURE
+				(int) PLN_PLUGIN_DEPOSIT_STATUS_LOCKSS_AGREEMENT
```

Comment:
Will this status check change break content that's already in the `pln_deposits` table from before the new version of the plugin?